server/preview.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const Eleventy = require('@11ty/eleventy');
+const path = require('path');
+const toMarkdown = require('@sanity/block-content-to-markdown');
+const sanity = require('../sanity_utils/sanityClient.js');
+const client = require('../sanity_utils/sanityClient.js');
+const serializers = require('../sanity_utils/serializers');
+
+const querySanity = async function (slug) {
+  const query = `*[_type == "post" && slug.current == '${slug}']`;
+  const result = await sanity.fetch(query);
```

Comment:
Could it be worth adding a timing log here to see if the bottleneck is latency between GAE and Sanity?